lib/cockpit-components-select.jsx

```diff
@@ -47,7 +47,10 @@ define([
             };
         },
         loseFocus: function(ev) {
-            this.setState({ open: false });
+            var self = this;
+            window.setTimeout(function() {
+                self.setState({ open: false });
```

Comment:
Seems like this will break if the control gets focus again within 100 ms. That's possible, especially programatically. Do you think we should just do what boostrap does and clear dropdowns like this:

https://github.com/twbs/bootstrap/blob/master/dist/js/bootstrap.js#L908